ruma-events/src/lib.rs

```diff
@@ -139,6 +139,21 @@ mod event_type;
 // that expect `ruma_events` to exist in the prelude.
 extern crate self as ruma_events;
 
+pub use ruma_events_macros::{
```

Comment:
Since it's probably very uncommon for these to be used, let's have a separate public `macros` module that re-exports the derive macros.